tests/ui-testing/playwright-tests/vrl_visualize.spec.js

```
@@ -0,0 +1,700 @@
+  await page
+    .locator("div.q-item")
+    .getByText(`${stream}`)
+    .first()
+    .click({ force: true });
+};
+test.describe(" visualize UI testcases", () => {
+  // let logData;
+  function removeUTFCharacters(text) {
+    // console.log(text, "tex");
+    // Remove UTF characters using regular expression
+    return text.replace(/[^\x00-\x7F]/g, " ");
```

Biome

[error] Unexpected control character(s) in regular expression: \x00

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)